themes/default/content/blog/managing-secrets-with-pulumi/index.md

```diff
@@ -6,6 +6,10 @@ authors: ["matt-ellis"]
 tags: ["features","Security"]
 ---
 
+{{% notes type="warning" %}}
```

Comment:
I don't think warning is quite what we want here. Maybe we should try the `info` type instead of warning? 

![image](https://github.com/pulumi/pulumi-hugo/assets/2553171/9c669b4c-4c72-4aa1-aad6-9a2ccafa5c38)

Reply:
updating